code/lib/manager-api/src/index.tsx

```diff
@@ -520,7 +520,6 @@ export { addons } from './lib/addons';
 /**
  * We need to rename this so it's not compiled to a straight re-export
  * Our globalization plugin can't handle an import and export of the same name in different lines
```

Comment:
this comment still shows up to users on hover even though it should be internal, which is pretty unfortunate.

Reply:
OK, I can change it to not be shown.